pontoon/teams/templates/teams/widgets/team_list.html

```diff
@@ -39,7 +39,7 @@ <h4>
       {% if chart.total_strings %}
         {{ ProgressChart.span(chart, chart_link, link_parameter, has_param) }}
       {% else %}
-        <span class="not-ready">Not synced yet</span>
+        <span class="not-ready">No projects enabled yet</span>
```

Comment:
This is not OK, because this template is also used by the project dashboard, e.g. https://pontoon.mozilla.org/projects/firefox/. And we want to display "Not synced yet" when a new locale is added to the project and hasn't synced with the VCS yet (which exposes strings for localization for that particular locale within the project).

So you'll need another "else if" here that will figure out which title to use.

Reply:
Changed. My variant has a new variable boolean for_teams_page. It's true if only /pontoon/teams/templates/teams/teams.html call it. 

Another idea is to give text variable to TeamList.item() with exact message.

Reply:
Good job!

Could you indeed please use something like `not_ready_message` instead? The default value could be `Not synced yet` and the `teams.html` template will then override it.

Reply:
@mathjazz Done with not_ready_message. Please, confirm that it's correct.